framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -319,61 +350,71 @@ public int runTests() throws Exception {
      
 
 
+
       
       if (cmdParam.iterations > 1) {
-      LOG.info(LINE_BREAK);
-      LOG.info(String.format("\nCompleted %d iterations.\n  Passing tests: %d \n  Random failures: %d \n  Execution Failures: %d\n  VerificationFailures: %d" +
+      	LOG.info(LINE_BREAK);
+      	LOG.info(String.format("\nCompleted %d iterations.\n  Passing tests: %d \n  Random failures: %d \n  Execution Failures: %d\n  VerificationFailures: %d" +
     	  "\n  Timeouts: %d\n  Canceled: %d", i-1, totalPassingTests, totalRandomFailures,totalExecutionFailures, 
     	  totalVerificationFailures, totalTimeoutFailures, totalCancelledFailures));
-    //}
-      if(finalRandomFailures.size()>0){
-      	LOG.info(LINE_BREAK);
-      	LOG.info("Random Failures");
-      	LOG.info(LINE_BREAK);
-      	for(DrillTest test : finalRandomFailures){
-      	 	LOG.info(test.getInputFile());
+      	if(finalRandomFailures.size()>0){
+      		LOG.info(LINE_BREAK);
+      		LOG.info("Random Failures:");
+      		LOG.info(LINE_BREAK);
+      		for(DrillTest test : finalRandomFailures){
+      	 		LOG.info(test.getInputFile());
+      		}
       	}
-      }
-      if(finalExecutionFailures.size()>0){
-      	LOG.info(LINE_BREAK);
-      	LOG.info("Execution Failures");
-      	LOG.info(LINE_BREAK);
-      	for(DrillTest test : finalExecutionFailures){
-      		 LOG.info(test.getInputFile());
+      	if(finalExecutionFailures.size()>0){
+        	LOG.info(LINE_BREAK);
+      		LOG.info("Execution Failures:");
+        	LOG.info(LINE_BREAK);      
+      		for (DrillTest test : finalExecutionFailures) {
+      			if(test.getExpectedFile()!=""){
+                         	LOG.info(test.getInputFile());
+                 	}
+		}
+		if(cmdParam.runFailed == true){
+			LOG.info("Execution Failures with name errors:");
+      		}
+		for (DrillTest test : finalExecutionFailures) {
+      			if(test.getExpectedFile()==""){
+                         	LOG.info(test.getInputFile());
+                 	}
+		}
       	}
-      }
-      if(finalVerificationFailures.size()>0){
-      	LOG.info(LINE_BREAK);
-      	LOG.info("Verification Failures");
-      	LOG.info(LINE_BREAK);
-      	for(DrillTest test : finalVerificationFailures){
-      	 	LOG.info(test.getInputFile());
+      	if(finalVerificationFailures.size()>0){
+      		LOG.info(LINE_BREAK);
+      		LOG.info("Verification Failures");
+      		LOG.info(LINE_BREAK);
```

Comment:
Remove redundant LINE_BREAK here. Let each section be separated by one LINE_BREAK